Clean up all Vale errors (#1643)
ngrok/ngrok-docs: CONTRIBUTING.md

# Adding New Content

> - If you're adding new content (for example, a guide, example), create a new mdx file within the appropriate subdirectory of the docs directory.

Author: this one is a pretty awkward find+replace 😅 ooh and also we don't have the `docs` dir anymore

```suggestion
- To add new content (such as a guide or an example), create a new MDX file in the appropriate directory.
```
